packages/components/components/elvis-tooltip/package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
     "@elvia/elvis-component-wrapper": "^4.1.3",
     "@elvia/elvis-toolbox": "^11.0.0",
```

Comment:
Is there anything used from the `toolbox` in this component now? 🙂 We can evaluate this during refactor, and hopefully we can remove a couple of couplings to the toolbox

Reply:
It still uses the `useSlot`-hook, `IconWrapper`-component, and `IconButton`, so it's not quite there yet!